components/compositing/compositor.rs

```diff
@@ -664,8 +664,8 @@ impl<Window: WindowMethods + ?Sized> IOCompositor<Window> {
                 self.waiting_for_results_of_scroll = true;
 
                 let mut txn = Transaction::new();
-                txn.scroll_node_with_id(point, scroll_id, ScrollClamping::ToContentBounds);
-                txn.generate_frame(0);
+                txn.scroll_node_with_id(point, scroll_id, ScrollClamping::NoClamping);
+                txn.generate_frame(200);
```

Comment:
Is this argument to `generate_frame` arbitrary? If not, maybe we can comment why this value was chosen?

Reply:
Sorry, this argument is currently unused. I modified it in the process of debugging, but I'll revert this change.

Reply:
This should be removed now. Thanks for pointing it out.